index.d.ts

```diff
@@ -0,0 +1,121 @@
+export type AnimationDirection = 1 | -1;

```

Comment:
There was no `.gitignore` rule for this file in `master/main` branches. I wasn't sure if I should add one or not so I didn't.

Reply:
I honestly don't know about this file, either.